modules/client/examples/01-client-token/11-get-token.ts

```diff
@@ -2,23 +2,23 @@
 ### Loading a plugin's token details
 */
 import {
-  ClientErc20,
+  ClientToken,
   Context,
   ContextPlugin,
-  Erc20TokenDetails,
+  TokenTokenDetails,
```

Comment:
That doesn't feel right. Should it simply be `TokenDetails`? or `PluginTokenDetails`?